### PR TITLE
feat: enrich Telegram status with project and agent context

### LINKED ISSUE #353: Enhance Telegram /status with project context and live agent activity
## Context
The Telegram bridge `/status` command currently replies with only the mapped session display string (e.g. `Current session: <title> (<id>)`). This is not enough to understand where commands are running or whether the agent is actively working.

From current implementation:
- `/status` handler in `shared/telegram-bridge.ts` only sends current session text.
- Bridge already tracks runtime activity via `statusBySession` from `session.status` events (`busy`/`retry`/`idle`).
- Bridge config and source mapping already include project directory context (`directory` per source/default source).

## Goal
Make `/status` actionable by showing:
1. current mapped session,
2. current project context (source + directory/project),
3. current agent activity state (idle vs working/retrying),
4. sub-session/sub-agent activity visibility (at minimum, whether any are active/known).

## Proposed Output (example)
```text
Session: Research thread (abc123)
Project: /workspace/foo (source: default)
Agent: busy (working)
Subsessions: 2 active
```

If some data is unavailable, return explicit placeholders (e.g. `Project: unknown`, `Subsessions: none`) instead of omitting fields.

## Files to Modify
- `shared/telegram-bridge.ts`
  - Extend `/status` command response formatting.
  - Add helper(s) to derive project context from current session/source mapping.
  - Use tracked status map and/or fresh `/session/status` lookup to report current activity.
  - Add/derive sub-session activity signal if available from existing runtime/session APIs.
- `app-prefixable/tests/telegram-bridge.test.ts`
  - Add/update tests for `/status` multiline output including project + agent state.
  - Add tests for fallback behavior when project/status/subsession data is unavailable.
- `README.md`
  - Update Telegram command docs so `/status` description matches new behavior.

## Acceptance Criteria
- `/status` returns a structured multi-line response, not just session id/title.
- Response includes current project context (directory or equivalent project identifier) and source id when relevant.
- Response includes live agent status for the mapped session (`idle`, `busy`, or `retry`-style state).
- Response includes sub-session/sub-agent activity information (count/state), or an explicit `none/unavailable` indicator when no data exists.
- Existing behavior for invalid mapping/unavailable source remains correct.
- Unit tests cover:
  - mapped session + project + busy state,
  - idle state,
  - retry state,
  - missing metadata fallbacks,
  - formatting normalization/truncation compatibility with current status/sessions tests.
- `README.md` command list describes the richer `/status` output.

## Notes for Implementation
- Prefer existing source/session helpers in `shared/telegram-bridge.ts` (`sourceForSessionRef`, `formatSessionDisplay`, `statusBySession`) to avoid duplicate mapping logic.
- Keep response under Telegram message limits and reuse current truncation helpers.

### PR DESCRIPTION
**Pull request by geier**
## Summary
- Expand /status from a single-line session mapping into a structured multiline response.
- Add project/source context, live agent state (idle/busy/retry/unknown), and subsession activity summaries with explicit fallbacks.
- Cover the new formatting and fallback behavior with focused bridge tests and update README command docs.

## Testing
- bun test app-prefixable/tests/telegram-bridge.test.ts (pass)
- bun run build (in app-prefixable) fails locally because esbuild is not installed in this worktree.

Closes #353

### CODE REVIEW

File: shared/telegram-bridge.ts [hunk 1]
```diff
@@ -1591,6 +1591,124 @@ type SessionLookup = {
...
+  const res = await fetch(url, {
+    method: "GET",
+    signal: AbortSignal.timeout(8_000),
+  }).catch(() => undefined)
+  if (!res?.ok) {
+    return { bySession: new Map(), reachable: false }
+  }
+  const payload = await res.json().catch(() => undefined)
+  if (!payload || typeof payload !== "object") {
+    return { bySession: new Map(), reachable: true }
```
[the remark this answers is not in the document]
Comment (geier) [reply shared/telegram-bridge.ts]: Fixed in c4d888a. `readSessionStatusSnapshot` now treats malformed/non-object payloads as unreachable (`reachable: false`), so `/status` falls back to `Agent: unknown` instead of incorrectly reporting idle. Added regression coverage in telegram-bridge tests.

File: shared/telegram-bridge.ts [hunk 2]
```diff
@@ -1591,6 +1591,124 @@ type SessionLookup = {
...
+    url.searchParams.set("directory", config.directory)
+  }
+  const res = await fetch(url, {
+    method: "GET",
+    signal: AbortSignal.timeout(8_000),
+  }).catch(() => undefined)
+  if (!res?.ok) return { text: "unavailable" }
+  const payload = await res.json().catch(() => undefined)
+  const sessions = Array.isArray(payload) ? payload : []
+  const ids = sessions
```
[the remark this answers is not in the document]
Comment (geier) [reply shared/telegram-bridge.ts]: Fixed in c4d888a. `readSubsessionSummary` now returns `unavailable` when JSON parsing fails or the payload is not an array, preserving `none` only for a real empty array. Added tests for both invalid JSON and non-array payload edge cases.
